content/300-guides/200-deployment/999-production-troubleshooting.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Production troubleshooting
+metaDescription: Troubleshooting Prisma Migrate issues in production environments.
+---
+
+<TopBlock>
+
+The following guide describes issues relating to Prisma Migrate that can occur in production, and how to resolve them.
+
+</TopBlock>
+
+## Failed migration
+
+A migration might fail if:
+
+- You [modify a migration before running it](../../../concepts/components/prisma-migrate#customizing-migrations-to-prevent-data-loss) and introduce a syntax error
+- You add a mandatory (`NOT NULL`) column to a table that already has data
+- The migration process stopped unexpectedly
+- The database shut down in the middle of the migration process
+
+Each migration in the `_prisma_migrations` table has a `logs` column that stores the error.
+
+There are two ways to deal with failed migrations in a production environment:
+
+- Roll back, optionally fix issues, and re-deploy
+- Manually complete the migration steps and resolve the migration
+
+#### Option 1: Roll back and re-deploy
+
+The following example demonstrates how to roll back a migration, optionally make changes to fix the issue, and re-deploy:
+
+1. Roll back the migration - this updates the migration record in the `_prisma_migrations` table to register it as rolled back, allowing it to be applied again:
```

Comment:
```suggestion
1. Mark the migration as rolled back - this updates the migration record in the `_prisma_migrations` table to register it as rolled back, allowing it to be applied again:
```

I think it's important to avoid users thinking that this will roll back the migration.